Helpers/Actions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRCTray.Helpers
+{
+    static class Actions
+    {
+        internal const string Start = "start application";
+        internal const string OpenMenu = "open menu";
+        internal const string ClickStart = "click start";
+        internal const string EnterPullSecret = "enter pull secret";
+        internal const string ClickStop = "click stop";
+        internal const string ClickDelete = "click delete";
+        internal const string ClickOpenConsole = "click open web console";
+        internal const string OpenPreferences = "open preferences";
+        internal const string ApplyPreferences = "apply new preferences";
+        internal const string OpenStatus = "open status";
+        internal const string OpenAbout = "open about";
+        internal const string CopyOCLoginForDeveloper = "copy oc login for developer";
+        internal const string CopyOCLoginForAdmin = "copy oc login for admin";
+        internal const string Quit = "quit application";
+        internal const string ChangeKubernetesContext = "change Kubernetes context";

```

Comment:
We don't yet have any use for this action, but in the future

Reply:
Copied it to stay 'consistent'. It is currently unused, yes.
Let's create an issue after merge that refers to this line.